torch/csrc/prototypes/nestedtensor/init.cpp

```diff
@@ -0,0 +1,364 @@
+// THIS CODE IS AUTOGENERATED! DO NOT MODIFY!
+// The script is under
+// torch/prototypes/nestedtensor/codegen/tensor_list.py
+#include <torch/csrc/utils/pybind.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <ATen/ATen.h>
+
+#include <vector>
+#include <iostream>
+
+namespace torch {
+namespace prototypes {
+namespace nestedtensor {
+void nestedtensor_abs(std::vector<at::Tensor>& input1,
+                       std::vector<at::Tensor>& out) {
+  for (int64_t i = 0; i < input1.size(); i++) {
+    at::abs_out(out[i], input1[i]);
+  }
+}
+void nestedtensor_acos(std::vector<at::Tensor>& input1,
+                       std::vector<at::Tensor>& out) {
+  for (int64_t i = 0; i < input1.size(); i++) {
+    at::acos_out(out[i], input1[i]);
```

Comment:
Wasn't the plan to disallow broadcasting for this iteration? Shouldn't there be additional size checks?

Reply:
Nvm, I see them in the other code.